# HG changeset patch
# User Alexis Ballier <aballier@gentoo.org>
# Date 1236540650 -3600
# Node ID 044a503f894304328c991f6495f9e14c4c4e5ecb
# Parent  ba642133c0c8b4ece4cb4d2a98355afde7264900
Fix build with libavutil >= 50.0.0

PIX_FMT_RGBA32 was #defined to PIX_FMT_RGB32 since 2006.

diff -r ba642133c0c8 -r 044a503f8943 src/combined/ffmpeg/ff_video_decoder.c
--- a/src/combined/ffmpeg/ff_video_decoder.c	Sun Mar 08 16:33:02 2009 +0000
+++ b/src/combined/ffmpeg/ff_video_decoder.c	Sun Mar 08 20:30:50 2009 +0100
@@ -588,7 +588,7 @@
       img->width,
       this->bih.biHeight);
 
-  } else if (this->context->pix_fmt == PIX_FMT_RGBA32) {
+  } else if (this->context->pix_fmt == PIX_FMT_RGB32) {
           
     int x, plane_ptr = 0;
     uint32_t *argb_pixels;
@@ -1304,7 +1304,7 @@
 
 	  /* initialize the colorspace converter */
 	  if (!this->cs_convert_init) {
-	    if ((this->context->pix_fmt == PIX_FMT_RGBA32) ||
+	    if ((this->context->pix_fmt == PIX_FMT_RGB32) ||
 	        (this->context->pix_fmt == PIX_FMT_RGB565) ||
 	        (this->context->pix_fmt == PIX_FMT_RGB555) ||
 	        (this->context->pix_fmt == PIX_FMT_BGR24) ||
